activesupport/lib/active_support/multibyte/chars.rb

```diff
@@ -7,7 +7,7 @@
 
 module ActiveSupport # :nodoc:
   module Multibyte # :nodoc:
-    # Chars enables you to work transparently with UTF-8 encoding in the Ruby
+    # Chars enable you to work transparently with UTF-8 encoding in the Ruby
```

Comment:
The previous one is correct usage, its referring to the class here.

Reply:
Thanks!
I reverted it back!